src/vs/workbench/contrib/notebook/browser/contrib/find/notebookFindReplaceWidget.ts

```diff
@@ -474,6 +484,58 @@ export abstract class SimpleFindReplaceWidget extends Widget {
 
 		this._innerReplaceDomNode.appendChild(this._replaceBtn.domNode);
 		this._innerReplaceDomNode.appendChild(this._replaceAllBtn.domNode);
+
+		this._resizeSash = new Sash(this._domNode, { getVerticalSashLeft: () => 0 }, { orientation: Orientation.VERTICAL, size: 2 });
+
+		this._sashListener.add(this._resizeSash.onDidStart(() => {
+			this._resizeOriginalWidth = this._getDomWidth();
+		}));
+
+		this._sashListener.add(this._resizeSash.onDidChange((evt: ISashEvent) => {
+			let width = this._resizeOriginalWidth + evt.startX - evt.currentX;
+			if (width < NOTEBOOK_FIND_WIDGET_INITIAL_WIDTH) {
+				width = NOTEBOOK_FIND_WIDGET_INITIAL_WIDTH;
+			}
+
+			const maxWidth = this._getMaxWidth();
+			if (width > maxWidth) {
+				width = maxWidth;
+			}
+
+			this._domNode.style.width = `${width}px`;
+
+			if (this._isReplaceVisible) {
+				this._replaceInput.width = dom.getTotalWidth(this._findInput.domNode);
```

Comment:
I believe that this is the width of the inner `findInput` that we need the width for to set the `replaceInput`. So instead of the whole dom element, it's the inner text input.